karma.conf.js

```diff
@@ -57,6 +57,11 @@ export default function configKarma(karma : Object) {
 
             'src/index.js': [ 'webpack', 'sourcemap' ],
             'src/**/*.js':  [ 'sourcemap' ]
+        },
+
+        coverageReporter: {
```

Comment:
I'm messing around with collecting coverage from karma and having github actions upload the report to codecov.io (ex: https://codecov.io/github/paypal/paypal-js). It's still not working. Still hacking away at it

Reply:
I got code coverage working! https://codecov.io/gh/paypal/paypal-checkout-components/tree/e49924c26ed11539c3ae2bde68b7e31463e0cac8/src. Once this gets merged, the readme badge should start working too.

<img width="1164" alt="Screen Shot 2020-12-07 at 4 56 58 PM" src="https://user-images.githubusercontent.com/534034/101415302-469b1b80-38ad-11eb-8f05-495414f5f422.png">